i18n/chs/out/commands.i18n.json

```diff
@@ -1,4 +1,5 @@
 {
   "autolispext.help.commands.openWebHelp": "无法加载 webHelpAbstraction.json 文件",
```

Comment:
@hualin-whl 
I remember that we only need to add new resource string for enu, and localization team will update all the files for other languages, am I right?

Reply:
For the i18 folder, your question to Hualin is exactly how it was described to me. I just made that app to populate the empty entries for the other languages so it was more clear what they needed to back fill.

For the root language files, the gulp  process automatically adds the missing entries referenced in the package.json, but it propagates the available version (english) into all the files. Which is a little less intuitive than empty string, but should still be vividly clear what needs back filled.

Reply:
Hi Josh, I checked our localization process. We (feature developers) only need to update files for "enu". We can save the effort here.

Reply:
I agree here! Sorrry if I introduced some confusion here!
So for the pre-release extension we only support the ENU version. This is a clean and neat desicion.